### DISCUSSION TITLE
Transit in level editor

### DISCUSSION THREAD

**Discussion by DutchBaller** [MattFiler/LegendPlugin#60] [Q&A]
So will we be able to place transits in the level editor? Maybe transits that transport you to other levels or the one from the HAB_CORPORATEPENT where it is a two way transit that you can travel between two stops. (Technically a one way I just say two way since there is no level transition)

**Comment by ghramsey** [comment]
You can technically do this now.  If you were to rewrite where a tranist goes.  The switch level settingjust points to a new mission number.
You'd also need to be mindful of the 'spawn point' or 'spawn position' (I forget what exactly it's called) parameter since it determines the spawn point on the chosen map.

The need to change out the in-game signage and such though would be kinda minds boggling and I think it's graphics not just text
which complicates things.  But in technicality the ability to chose where a transit goes is possible with existing tools as of 2023.

**Reply by MattFiler** [reply]
You can edit the transit signs by replacing the materials on the signs when you instance the models :)

**Reply by ghramsey** [reply]
Thank you for the input.  

It'd still be a daunting task.  Too complex for my very limited skills.  
Hahah. This is an almost 1 yr old thread....
